subsys/mpsl/fem/mpsl_fem.c

```diff
@@ -300,33 +300,6 @@ static int fem_nrf21540_gpio_configure(void)
 #endif
 
 #if DT_NODE_HAS_STATUS(MPSL_FEM_SPI_IF, okay)
-	err = inactive_pin_configure(
```

Comment:
Please add a comment here that the reason for only the CS pin being configured is that the other pins may be used to communicate with other SPI peripherals.